src/main/java/page_objects/BasePage.java

```diff
@@ -8,14 +8,17 @@
 
     //Locators
     private final By tabLogin = By.cssSelector("li a[href='/Account/Login.cshtml']");
+    private final By ContactTab = By.cssSelector("li a[href='/Page/Contact.cshtml']");
```

Comment:
It should be tabContact
Please check format anytime you create a new variables or method, make sure they are the same

Reply:
I've already fixed it. Please help me to check, thank so much.